homeassistant/components/cover/__init__.py

```diff
@@ -38,6 +38,15 @@
     'garage',        # Garage door control
 ]
 
+SUPPORT_OPEN = 1
+SUPPORT_CLOSE = 2
```

Comment:
Why would we separate open and close? I don't think there will ever be a use case that has no open and close.

Even if a device does not have open and close specific but has set position, they could mimick open and close by setting position to fully open or fully closed. So I think that we can remove those.